vhost-user-backend/Cargo.toml

```diff
@@ -16,7 +16,7 @@ postcopy = ["vhost/postcopy", "userfaultfd"]
 libc = "0.2.39"
 log = "0.4.17"
 userfaultfd = { version = "0.8.1", optional = true }
-vhost = { path = "../vhost", version = "0.13.0", features = ["vhost-user-backend"] }
+vhost = { path = "../vhost", version = "0.14.0", features = ["vhost-user-backend"] }
```

Comment:
Please move this change in the previous commit, otherwise we will break the bisectability.